content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,183 @@
+Finally, click :guilabel:`Save` to finish creating the attribute.
 
-You can also add a different barcode and internal reference to the
-variant.
+.. tip::
+   Attributes can also be created directly from the product template by adding a new line and
+   typing the name into the :guilabel:`Variants` tab.
 
-When you have entered all the specifications of the variant, click on
-**Save**.
+After an attribute is added to a product, a new tab appears on the attribute's page called
+:guilabel:`Related Products`. This tab lists every product in the database that is currently using
+the attribute.
 
-Managing Product Variants
-=========================
+Creating product variants
+-------------------------
 
-Introduction
-------------
+After creating an attribute, use the attribute to create a product variant by going to
+:menuselection:`Sales --> Products --> Products`. Then, select an existing product and click
+:guilabel:`Edit`, or create a new product by clicking :guilabel:`Create`.
 
-The examples below are all based on this product template that has two
-variant attributes :
+The :guilabel:`Variants` smart button at the top of the product template indicates the number of
+currently configured variants on the product.
 
--  T-Shirt B&C
+To add a new variant, click on the :guilabel:`Variants` tab, then click on :guilabel:`Add a line`
+to add any attributes and values. When all the attributes and values have been added, click
+:guilabel:`Save`.
 
-   -  Color: Red, Blue, White
+.. tip::
+   Similar product variant creation processes are accessible through the Purchase, Inventory, and
+   eCommerce applications.
 
-   -  Size: S, M, L, XL, XXL
+Managing product exclusions
+---------------------------
 
-Managing combination possibilities
-----------------------------------
+The following examples are all based on a product template that has two attributes:
 
-By default, with the above product template, you get 15 different
-products (3 colors, 5 sizes). If the XXL size only exists for red and
-blue t-shirts, you can deactivate the white product variant.
+- T-Shirt
 
-To do this, click on the **Variants** button, select the XXL, White
-T-shirt. From the product form, uncheck the **Active** box of the T-shirt
-White, XXL.
+  - Color: Blue, Red, White
+  - Size: S, M, L, XL, XXL
 
-.. image:: variants/variant04.png
-   :align: center
+With the above product template, there are 15 different t-shirt variants in three different colors
+and five different sizes. If the white t-shirts are not available in the XXL size, then that
+variant can be deactivated.
 
-.. tip::
+To deactivate a particular product variant, go to :menuselection:`Sales --> Products --> Products`
+and select the relevant product. Next, click on the :guilabel:`Configure Variants` button, open the
+relevant product variant attribute value form (in this example, the :guilabel:`White Color`
+attribute), and then click on :guilabel:`Edit`. In the :guilabel:`Exclude for` section, click
```

Comment:
I would line break to another paragraph between `` ...then click on :guilabel:`Edit`. `` and `` In the :guilabel:`Exclude for` section... ``